src/debug/HyprNotificationOverlay.cpp

```diff
@@ -81,25 +70,25 @@ CBox CHyprNotificationOverlay::drawNotifications(CMonitor* pMonitor) {
     float                 offsetY  = 10;
     float                 maxWidth = 0;
 
-    const auto            SCALE = pMonitor->scale;
-
+    const auto            SCALE   = pMonitor->scale;
     const auto            MONSIZE = pMonitor->vecTransformedSize;
 
-    cairo_text_extents_t  cairoExtents;
-    int                   iconW = 0, iconH = 0;
+    static auto           fontFamily    = CConfigValue<std::string>("misc:font_family");
+    auto                  iconBackendID = get_iconBackendID(*fontFamily);
+
+    PangoLayout*          layoutIcon = pango_cairo_create_layout(m_pCairo);
+    PangoLayout*          layoutText = pango_cairo_create_layout(m_pCairo);
+    PangoFontDescription* pangoFD    = pango_font_description_new();
 
-    cairo_select_font_face(m_pCairo, "Sans", CAIRO_FONT_SLANT_NORMAL, CAIRO_FONT_WEIGHT_NORMAL);
+    pango_font_description_set_family_static(pangoFD, (*fontFamily).c_str());
+    pango_font_description_set_style(pangoFD, PANGO_STYLE_NORMAL);
+    pango_font_description_set_weight(pangoFD, PANGO_WEIGHT_NORMAL);
 
```

Comment:
Here I'm trying to get the latest `font_family` settings and apply it to the new draw, however it does not seem to use the configured font, even with a restart of Hyprland. As As it's setup just like `CHyprError::createQueued()`, which works well for dynamic font update, but this does not work, I can't figure it out, would you like to have look at it? the single commit: 260fe70960aa27412d555715024e14f4cf7679e6

Reply:
I don't think I've ever used that func to set a font family, maybe check out how hyprlock renders text? it uses pango too.